lib/welcome_screen.dart

```diff
@@ -0,0 +1,95 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:transportation_app/login_screen.dart';
+import 'package:transportation_app/register_screen.dart';
+import 'package:transportation_app/widgets/customized_elevatedbutton.dart';
+import 'package:transportation_app/widgets/customized_text.dart';
+
+class WelcomeScreen extends StatelessWidget {
+  const WelcomeScreen({Key? key}) : super(key: key);
+
+  //Hocam burada Anasayfadaki resmi tanımladım,arkaplanı ekrana sığdırdım.
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: Container(
+        height: MediaQuery.of(context).size.height,
+        width: MediaQuery.of(context).size.width,
+        decoration: const BoxDecoration(
+          image: DecorationImage(
+            image: AssetImage("images/welcomescreen_container.png"),
+            fit: BoxFit.cover,
+          ),
+        ),
+
+        //Burada Logoyu ekledim.
+
+        child: Center(
+          child: Column(
+            mainAxisAlignment: MainAxisAlignment.end,
+            children: [
+              Padding(
+                padding: const EdgeInsets.all(8.0),
+                child: Row(
+                  mainAxisAlignment: MainAxisAlignment.center,
+                  children: [
+                    SvgPicture.asset("images/transportation_icon.svg"),
+                  ],
+                ),
+              ),
+
+              //Burada logo altına gelen yazıyı ekledim.
+
+              Padding(
+                padding: const EdgeInsets.all(8.0),
+                child: customizedText(
+                    text: "Let's transfer your package \n"
+                        "in a comfortable and easy way !",
+                    textcolor: const Color(0xffEEEEFE)),
+              ),
```

Comment:
Bu \n yöntemi çok sağlıklı bir yöntem değil. Bunun yerine textAlign parametresini customized text'in içinde parametre olarak ayarlarsan yazının konumunu ayarlayabilirsin veya parametre olarak kullanmak istemezsen Flexible ya da Expanded widgetlarını kullanabilirsin.

Reply:
Hocam bunu da Expanded olarak değiştirdim.